Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -395,6 +396,41 @@ def test_pascalToSpace():
         assert pascalToSpace(s) == expected, 'Error on {} != {}'.format(pascalToSpace(s), expected)
 
 
+def test_safe_load_json():
+    valid_json_str = '{"foo": "bar"}'
+    expected_valid_json_result = {u'foo': u'bar'}
+    assert expected_valid_json_result == safe_load_json(valid_json_str)
+
+
+def test_datetime_to_string():
+    datetime_obj = datetime.now()
+    datetime_str = datetime_to_string(datetime_obj)
```

Comment:
remove this test, no need for that. instead add extra test for safe_load_json that contains date,